bin/create_svg_manifest.js

```diff
@@ -0,0 +1,32 @@
+const glob = require('glob');
+const _ = require('lodash');
+const fs = require('fs'); // eslint-disable-line
```

Comment:
It was throwing errors because of my local npm or something, I think. I can revisit it today tho